### DISCUSSION TITLE
Contents for a how-to guide?

### DISCUSSION THREAD

**Discussion by quantumjot** [quantumjot/btrack#323] [Ideas]
We get many requests for advice on what the hyperparameters mean, how to set and tune them. If we were to write a "how-to" guide, what would people like to see? Feel free to post your ideas here!

**Comment by marabuuu** [comment]
Hey @quantumjot,

Nice that you want to write a how-to guide! Are you planning to append it to the README of the plugins documentation? :) 

Here some ideas 

_General:_ 
- What are motion and hypothesis model?
- Which scientific questions would correspond to tuning the one or the other?
- What is the best way of parameter tuning to keep an overview and be time effective (e.g. which parameters to start with best, which ones are less important and can be done later)

_For the napari plugin:_
- How can I differentiate between 2D timelapse and 3D timelapse?
- What do the abbreviations under Motion model and Hypothesis model mean? When do I need to change them? For which applications 
   does it make sense to change them?

Best,
Mara

**Comment by markdane** [comment]
I expect I and others would get great benefit from a how-to guide on tuning the motion and hypothesis models. Thanks again for creating this package. Some of my questions are:

Our HCC1143 cells ball up before dividing, after division they spread out and look fairly similar image to image. How do we optimize the models so that cells and tracklets are connected before division and progeny cells are connected to their ancestor after division? Are you having good experience with vision features such as intensity and morphology?

I assume the parameters in the json configuration file are overwritten by configuration changes done in a script after calling tracker.configure_from_file. Maybe just a note to confirm that?

What is the expected performance difference between cell_config_flat.json and cell_config.json?

In addition to review of the lineage trees, do you recommend any specific QA metrics or methods to evaluate hyper parameters when tuning? I'm hoping to avoid manually creating a ground truth dataset of my images.

cheers,
Mark

**Comment by markdane** [comment]
Can you please point me to explanations on or help how to use the lambda values in the hypothesis model. The optimization time and results seem sensitive to the lambda_link value in particular.

I am currently getting short cell lives that are shown dividing too quickly. About 20% of my cells are labeled as lazy initializations and about 5% are lazy terminations.

thanks!
